LEAD_MAINTAINER_PROTOCOL.md

```diff
@@ -1,51 +1,72 @@
-## Tech Leads and Lead Maintainers
+## Repo Lead Maintainer Protocol
 
-> This section is a formalization of the proposal and discussion around the [Module Lead Maintainers - Sharing the Responsibility over the IPFS module base](https://github.com/ipfs/pm/issues/600) proposal.
+> This section is a formalization of the proposal and discussion around the [Module Lead Maintainers - Sharing the Responsibility over the IPFS module base](https://github.com/ipfs/pm/issues/600) proposal, originally desined for the JS Ecosystem within the IPFS Project.
 
-**Currently, this Protocol is only being executed by the JS ecosystem of the IPFS Project.**
+## Motivation
 
-We have two types of leads in the JavaScript project ecosystems, the Tech Lead and the Module Lead Maintainer. A brief description of both roles is:
+We develop the IPFS Project as an Asynchronous and Distributed Organization. In order to operate at a global scale, written documentation, curation and improving its discoverability needs be at the core of our competencies. Having clear and lean docs that are well maintained can avoid a misdirection or misinformation, saving hours in productivity and making our contributors happier.
 
-- A Tech Lead directs the development of an entire ecosystem of modules (i.e js-ipfs, js-libp2p, js-ipld and js-multiformats), it has a complete understanding of the stack, the IPFS project, its goals and participates actively in the ROADMAP planning.
-- A Lead Maintainer is a contributor that has shown extraordinary ability to contribute to the project and willingness to make the project better by taking the responsibility of stewarding one of its modules forward.
+To plant the idea of how important this written communiaction is, here is an thought example: Imagine that you spend 30 minutes refactoring a README of a repo, fixing broken links, making sure that the information is accurate and so on and in the end, what you achieve is that the next person will be able to save 1 minute of their time searching for the info they need. You might think "whoa, I just spend 30 minutes for an individual to save 1? Maybe this was not worth it at all". This thinking is incorrect, because within a project of 3500+ contributors, you are saving time for thousands. Some of those people might not have ever found the information if not for your work (or may have left the repo confused), which magnifies your contribution even more.
```

Comment:
Some typos :)
```suggestion
To plant the idea of how important this written communication is, here is a thought example: Imagine that you spend 30 minutes refactoring a README of a repo, fixing broken links, making sure that the information is accurate and so on and in the end, what you achieve is that the next person will be able to save 1 minute of their time searching for the info they need. You might think "whoa, I just spend 30 minutes for an individual to save 1? Maybe this was not worth it at all". This thinking is incorrect, because within a project of 3500+ contributors, you are saving time for thousands. Some of those people might not have ever found the information if not for your work (or may have left the repo confused), which magnifies your contribution even more.
```